wolfssl-3.9.6/README.md

```diff
@@ -0,0 +1,139 @@
+Finds a SEGV and three heap overflow errors in `src/ssl.c`
+in [wolfSSL v.3.9.6](https://www.wolfssl.com/wolfSSL/Home.html).
```

Comment:
How quickly does a fuzzing engine (AFL? libFuzzer?) find these bugs? 
We are mostly interested in benchmarks that take minutes, hours, or days -- not seconds.